# Contains a PathPointWithLaneId path and left and right bound.
std_msgs/Header header
autoware_auto_planning_msgs/PathPointWithLaneId[] points
geometry_msgs/Point[] left_bound
geometry_msgs/Point[] right_bound

